build_html.sh

```diff
@@ -54,7 +55,7 @@ done
 rm -r -f */temp.ipynb
 
 # download additional js to be included
-curl -O https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js
+curl -O https://cdn.plot.ly/plotly-1.57.0.min.js
```

Comment:
Why we need download .js file manually?

Reply:
For D2L website generation, all JS will be found and load locally. Then append to `d2l.js`.